packages/edit-post/src/components/preferences-modal/enable-custom-fields.js

```diff
@@ -39,8 +39,7 @@ export function CustomFieldsConfirmation( { willEnable } ) {
 				) }
 			</p>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
 				className="edit-post-preferences-modal__custom-fields-confirmation-button"
```

Comment:
Another classname to remove while we're here. The last usage of this one was removed a while ago in #35369.

You might need to update some snapshots if you remove it.